src/lib/testFlags.ts

```diff
@@ -47,6 +47,10 @@ class TestFlags {
   get referrer() {
     return this.queryParams.utm_source;
   }
+
+  get ohlc() {
```

Comment:
unclear what `ohlc` flag means. Does this mean use price but show orderbook to fill in 0 trade candles? Some descriptor here and on the TradingView toggle would be helpful.

Reply:
yup that's accurate! will add a comment. Proposal for when we implement the toggle is to have a tooltip which says the following: `Enable or disable OHLC data via mid-price. Disabling this will only display OHLC data from historical trades`